src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int side;
+    private int upSide;
+    private int downSide;
+    private final int halfPerimeter = (2 * side + upSide + downSide) / 2;
```

Comment:
`halfPerimeter` will be always `0`.
Remove this variable and move calculations directly in `getArea` method